stories/demos/CommonFeatures.tsx

```diff
@@ -13,6 +13,28 @@ const toolbarClassname = css`
   margin-bottom: 8px;
 `;
 
+const dialogContainerClassname = css`
```

Comment:
> The ::backdrop CSS pseudo-element can be used to style behind a <dialog> element when the dialog is displayed with HTMLDialogElement.showModal().

😞

Reply:
Would that work? 🤔 
```tsx
ref={dialog => dialog.showModal()}
```

Reply:
Tried but getting ref errors. May be I will try some other time